src/nvim/getchar.c

```diff
@@ -2011,9 +2011,7 @@ static int look_in_typebuf(int *mapdepthp, int *keylenp,
   return -1;
 }
 
-// Returns 0 to continue, 1 to break, -1 to carry on in the loop.
-// If it's found a key, leaves the new value of "c" in "*cp"
-static int8_t get_key_from_user(int *cp, int *timedoutp, int *mode_deletedp,
+static int ins_esc_special_case(int *new_wcolp, int *new_wrowp, int *mode_deletedp,
```

Comment:
Even for static functions, I very much appreciate a comment on what the function is supposed to accomplish. For me, it's not necessary to fully comment out the variables with name and in formal setting.

```
// Does A. Puts X in new_wcolp and Y in new_wrowp. mode_deletedp is Z after ...
static int ins_esc_special_case()
```